File: sum_of_oe/sum.c
#include<stdio.h>

int main()
{
int n,sum_o=0,sum_e=0;
printf("n=\n");
scanf("%d",&n);

for(int i=0;i<=n;i++)
{
  i%2==0 ? (sum_e += i) : (sum_o += i);
}
printf("sum_e=%d\n",sum_e);
printf("sum_o=%d\n",sum_o);

return 0;
}

File: sum_of_oe/sum.py
#!/usr/bin/python

n=int(input("n="))
sum_o=0
sum_e=0
#n=n+1
#for i in range(0,n+1)
#for i in range(n+1)
for i in range(n+1):
  if(i%2==0):
    sum_e += i
  else:
    sum_o += i

print "sum_e=" +str(sum_e)
print "sum_o=" +str(sum_o)
